src/modules/preview-site/components/create-preview-button.tsx

```diff
@@ -33,7 +36,10 @@ export function CreatePreviewButton( { onClick, selectedSite }: CreatePreviewBut
 		snapshotCreationBlocked ||
 		isOverLimit;
 
-	const siteArchivingMessage = __(
+	const currentSiteArchivingMessage = __(
+		"This site's preview is being created. Please wait for it to finish before creating another."
```

Comment:
Maybe we could consider slightly different wording:

```suggestion
		"A preview of this site is being created. Please wait for it to finish before creating another."
```

Reply:
Thanks, I made some adjustments in https://github.com/Automattic/studio/pull/956/commits/3a47a119f66f60b78638dd6a54c67e660d27d657

Reply:
Thanks, Kat.